dev/ci/ci-common.sh

```diff
@@ -46,6 +46,40 @@ done
 
 mathcomp_CI_DIR="${CI_BUILD_DIR}/math-comp"
 
+# [git_download project dest] will download the given [ref] of [project]
+# and unpack it in [dest] if the folder does not exist already;
+# if it does, it will do nothing except print a warning
+# (this can be useful when testing make ci-* locally).
+# When the variable $CI is undefined, it uses git clone to perform the
+# download.
+git_download()
+{
+  local PROJECT=$1
+  local DEST=$2
```

Comment:
Is there some reason to have `DEST` different from `PROJECT`?

Reply:
Nope, I considered removing it but this means that:

https://github.com/coq/coq/blob/7ca59e2db97d78c511856375991400e44a6c1934/dev/ci/ci-bignums.sh#L12-L16

would become

```shell
git_download bignums

(cd "${CI_BUILD_DIR}/bignums" && make && make install)
```

Does this look good? If yes, I can do it.

Reply:
The capitalisation of some likely-temporary directory doesn't seem important.

Reply:
Ah yes, I was not really asking about the capitalization, more about the rest. But it seems you find it OK, so I will go for it.